### PR TITLE
added missing documentation about case_insensitive param in wildcard …

### PR DESCRIPTION
**Pull request by macohen**
…query

### Issues Resolved
N./A

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _query-dsl/query-dsl/term.md [hunk 1]
```diff
@@ -405,6 +405,7 @@ Feature | Behavior
 :--- | :---
 `*` | Specifies all valid values.
 `?` | Specifies a single valid value.
+`case_insensitive` | Parameter to indicate if the wildcard query should consider case. false to be case sensitive. true to be case insensitive. this is not required and will be case sensitive by default.
```
Comment (kolchfa-aws) [inline _query-dsl/query-dsl/term.md:408]:
```suggestion
`case_insensitive` | If `true`, the wildcard query is case insensitive. If `false`, the wildcard query is case sensitive. Optional. Default is `true` (case insensitive).
```

File: _query-dsl/query-dsl/term.md [hunk 2]
```diff
@@ -420,6 +421,22 @@ GET shakespeare/_search
   }
 }
 ```
+
+To search for terms with case sensitivity on:
```
Comment (kolchfa-aws) [inline _query-dsl/query-dsl/term.md:425]:
```suggestion
Use the following request for a case-sensitive term search:
```

File: _query-dsl/query-dsl/term.md [hunk 3]
```diff
@@ -420,6 +421,22 @@ GET shakespeare/_search
...
+```json
+GET shakespeare/_search
+{
+  "query": {
+    "wildcard": {
+      "speaker": {
+        "value": "H*Y",
+		"case_insensitive": false
```
Comment (kolchfa-aws) [inline _query-dsl/query-dsl/term.md:433]:
```suggestion
	"case_insensitive": false
```

**Review by kolchfa-aws — COMMENTED** [review]
A couple of tweaks. This is for all versions, correct?

**Review by natebower — APPROVED** [review]
LGTM

**Comment by macohen** [comment]
The related issue is https://github.com/opensearch-project/OpenSearch/issues/8711

**Comment by kolchfa-aws** [comment]
Closing this PR in favor of #4914

**Pull request closed without merging**